src/config/apiTypes.ts

```diff
@@ -0,0 +1,7 @@
+export type Error = {
+  errors: Record<string, string[]> | string[];
+};
+
+export type Response = {
+  data: Record<string, string | number>;
+};
```

Comment:
place this code lines on `~utils/types.ts` remember that if you are adding a types file elsewhere, should be named `types.ts` or `interfaces.ts`, also `interface` is better to define the shape of an object.
Particularly both interfaces are so general, i would keep `Error`, rename it to `ApiError`. But the `Response` interface is so general, should be typed to every service you have, not by using general object shape.